hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -802,8 +802,9 @@ public int getRange() {
     CERTIFICATE_REVOKATION_ERROR(16, "Error revoking the certificate", Response.Status.INTERNAL_SERVER_ERROR),
     CERTIFICATE_NOT_FOUND(17, "Could not find the certificate", Response.Status.BAD_REQUEST),
     CERTIFICATE_REVOKATION_USER_ERR(18, "Error revoking the certificate", Response.Status.BAD_REQUEST),
-    CERTIFICATE_SIGN_USER_ERR(19, "Error signing the certificate", Response.Status.BAD_REQUEST);
-
+    CERTIFICATE_SIGN_USER_ERR(19, "Error signing the certificate", Response.Status.BAD_REQUEST),
+    ENTERPRISE_FEATURE(20, "This feature is only availlable in the enterprise edition", Response.Status.BAD_REQUEST);
```

Comment:
@berthoug I feel this error code fits better undet the GenericErrorCode Enum, since we could re-use it for instances other than Security ones

Reply:
fixed